VACMS-5671: Add leadership flag.

ethanteague opened the pull request:
## Description
See #5671

## Testing done
Visual

## QA steps
 - As an administrator, go to `admin/config/system/feature_toggle` and visually verify the presence of FEATURE_CHANGE_LEADERSHIP_LINK

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

kevwalsh reviewed (changes requested):
Wrong flag dude. Maybe i should have said "feature toggle" not "feature flag" 

You want it to appear here https://pr5691-jszyzr5kvnhyxz3jvkvw5pyu3zy4kzzi.ci.cms.va.gov/admin/config/system/feature_toggle not https://pr5691-jszyzr5kvnhyxz3jvkvw5pyu3zy4kzzi.ci.cms.va.gov/admin/structure/flags

ethanteague commented:
> Wrong flag dude. Maybe i should have said "feature toggle" not "feature flag"
> 
> You want it to appear here https://pr5691-jszyzr5kvnhyxz3jvkvw5pyu3zy4kzzi.ci.cms.va.gov/admin/config/system/feature_toggle not https://pr5691-jszyzr5kvnhyxz3jvkvw5pyu3zy4kzzi.ci.cms.va.gov/admin/structure/flags

Oh good lord. I was looking at the ui, and thinking "wow, they've really changed this and added a lot of features since I last created a flag!" Apologies, @kevwalsh ! I added the flag (for real this time). Heads up: these toggles don't have their states stored in config (the idea being that we can enable / disable on a per environment basis), so we will need to check the box once it is deployed to prod.

kevwalsh reviewed (approved):
![Feature_Toggle_list___VA_gov_CMS](https://user-images.githubusercontent.com/643678/123468455-c9709300-d5bf-11eb-9f77-ae67276f5c80.png)

QA pass. Just waiting for tests to pass then we should merge, although it may be too late for today. I'll set a reminder for us to enable this flag at EOD Monday, then rebuild the demo environment for Facilities to do another round of QA in Tugboat.
